src/cli.js

```diff
@@ -111,7 +117,7 @@ shouldRebuildPromise
   })
   .then((x, beforeRebuild) => {
     return installNodeHeaders(argv.v, argv.d, null, argv.a)
-      .then(() => rebuildNativeModules(argv.v, argv.m, argv.w, null, argv.a, argv.c))
+      .then(() => rebuildNativeModules(argv.v, argv.m, argv.w, null, argv.a, argv.c, argv.id, argv.io, argv.log))
```

Comment:
The CLI tool didn't appear to work that way off the top of my head, the alias mapped to the shorthand 😢